changelogs/fragments/5542-adding-features-to-ilo_redfish_config.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - added new features for ilo_redfish_utils  (https://github.com/ansible-collections/community.general/pull/5542).
```

Comment:
I think this is mentioning the wrong module. (Also the fragment format is wrong as in the other PRs.)

Reply:
Changed to  "- ilo_redfish_config - add new features  (https://github.com/ansiblecollections/community.general/pull/5542)."